platforms/core-configuration/model-core/src/main/java/org/gradle/internal/isolated/IsolationScheme.java

```diff
@@ -238,8 +238,8 @@ public Object find(Type serviceType) throws ServiceLookupException {
                 if (serviceClass.isAssignableFrom(BuildServiceRegistry.class)) {
                     return allServices.find(BuildServiceRegistry.class);
                 }
-                if (serviceClass.isAssignableFrom(Problems.class)) {
-                    return allServices.find(Problems.class);
+                if (serviceClass.isAssignableFrom(InternalProblems.class)) {
+                    return allServices.find(InternalProblems.class);
```

Comment:
Thank you for looking into this @hegyibalint!

Reply:
Trivial solution, isn't it?!... 😂